upgrade/Stage_0_Prerequisites.md

```diff
@@ -73,6 +86,12 @@ after a break, always be sure that a typescript is running before proceeding.
          cp PATH_TO_DOCS_RPM /root/docs-csm-latest.noarch.rpm &&
          rpm -Uvh --force /root/docs-csm-latest.noarch.rpm
          ```
+         
+   - This step must be performed if an http proxy was set previously.
```

Comment:
The step above "Download the latest docs-csm" references a page where you've already added the set/unset steps so I don't think this set of "unsets" is needed here.